types/node/test/perf_hooks.ts

```diff
@@ -100,11 +100,7 @@ histogram.record(123);
 histogram.record(123n);
 histogram.recordDelta();
 
-/**
- * Now intelligence is working for the "add" function added
- * under interface RecordableHistogram
- * under perf_hooks.ts
- */
+//intelligence is working
```

Comment:
```suggestion
// intelligence is working
```
more readable, current one, infamously, is a style (missing space) that MS Visual Studio is using as a default (but not VSCode IMO)

Reply:
Ok, thank you!